README.md

```diff
@@ -1 +1,34 @@
-# Official docker-compose files repository
+# Docker-compose files to run the Kuzzle stack
+
+**Supported OS**
+
+![Debian Jessie](https://kuzzleio.github.io/kuzzle-build/setupsh-badges/debian-jessie.svg) ![Fedora](https://kuzzleio.github.io/kuzzle-build/setupsh-badges/fedora.svg) ![Ubuntu Artful](https://kuzzleio.github.io/kuzzle-build/setupsh-badges/ubuntu-artful.svg) 
+
+The docker-compose files in the `docker-compose` dir will allow you to run Kuzzle with optional SSL support.
+
+## The `setup.sh` installation helper
+
+Use the `setup.sh` script to perform a requirements check on your system and automatically start Kuzzle.
+The script will _not_ install any dependencies on your behalf, but will instead give you hints on how to do it.
+
+To run it, you don't need to clone this repository, you can just type
+
+```
+$ bash -c "$(curl https://raw.githubusercontent.com/kuzzleio/kuzzle-build/master/setup.sh)"
+```
+
+Or, you can clone this repository and run the script with options
+
+```
+$ ./setup.sh --no-run
+```
+
+only performs the system requirements checks and pulls the Docker images for the stack, without running it.
```

Comment:
proposal: `...and pulls a Kuzzle stack from Docker repositories, without running it.`